### PR TITLE
T16153

### PR DESCRIPTION
**Pull request by LiliC**
Reword some things as per ticket suggestion. Remove the old social bar lesson, as we don't have it anymore and add clock-format lesson.

@erikos 

https://phabricator.endlessm.com/T16153
https://phabricator.endlessm.com/T15374

### CODE REVIEW

File: showmehow/lessons.json [hunk 1]
```diff
@@ -184,38 +184,38 @@
...
                 }
             },
-            "enable_social_bar": {
-                "task": "Now lets change it back to true",
+            "change_clock_format": {
+                "task": "Now lets change the 'clock-format' to the '24h' format. This is done in the same way as we just did to view the current clock format, but instead we will be using the 'set' command.",
                 "effects": {
                     "success": {
-                        "reply": "Okay, phew, Facebook is back",
+                        "reply": "Sucess! Look at the clock in the right corner of your screen, the format changed! You can also have a look at the chatbox application, and you might notice that the clock format is different there as well. Amazing what a little bit of text can do, right?",
```
Comment (erikos) [inline showmehow/lessons.json]: "...timestamp format of the messages..." maybe that is more accurate.

File: showmehow/lessons.json [hunk 2]
```diff
@@ -184,38 +184,38 @@
...
                     "failure": {
-                        "reply": "Hmm, I thought you'd change the value of org.gnome.shell e'enable-social-bar' to false, but it is still true. Remember to change a gsettings value you can use:\n$ gsettings set KEY VALUE"
+                        "reply": "Thats not what we want. Try replacing '<our schema>' with 'org.gnome.desktop.interface' and 'clock-schema-key' with 'clock-format' in the following command `gsettings get <our-schema> <clock-schema-key>`."
                     }
                 }
             },
-            "enable_social_bar": {
-                "task": "Now lets change it back to true",
+            "change_clock_format": {
+                "task": "Now lets change the 'clock-format' to the '24h' format. This is done in the same way as we just did to view the current clock format, but instead we will be using the 'set' command.",
```
Comment (erikos) [inline showmehow/lessons.json]: As discussed let's make this independent of the current format set and rephrase to something like: "...there you see your current format - now try flipping it, to the other format...".

**Comment by LiliC** [comment]
@erikos done. Can you have another look. I am sure if the failure msg can be improved, but can't think of anything right now.

File: showmehow/lessons.json [hunk 3]
```diff
@@ -184,38 +184,38 @@
...
+                        "reply": "I want to know about the settings available for org.gnome.desktop.interface in sorted order? Can you list-keys on org.gnome.desktop.interface using gsettings and then pipe (|) the output to `sort`? Try:\n$ gsettings list-keys org.gnome.desktop.interface | sort"
                     }
                 }
             },
-            "disable_social_bar": {
-                "task": "Lets disable the social bar! The social bar's setting is 'enable-social-bar' and it is a `boolean` setting, meaning it is either 'true' or 'false'. I checked `gsettings help set` for you and it said you can use the `set` command to change a setting with gsettings [--schemadir SCHEMADIR] set SCHEMA[:PATH] KEY VALUE. Your job is to provide KEY and VALUE, those being 'enable-social-bar' and 'false' to `gsettings set`.",
+            "get_clock_format": {
+                "task": "Lets learn how to change the clock format using 'gsettings'. The two avaliable formats are '12h' or '24h'. Lets start by finding out what format is set in the 'clock-format' key, by using the `get` command. The format should be the following. `gsettings get <our-schema> <clock-schema-key>`.",
```
Comment (cosimoc) [inline showmehow/lessons.json]: Can you change `Lets` to `Let's` in this paragraph?

File: showmehow/lessons.json [hunk 4]
```diff
@@ -184,38 +184,38 @@
...
                 "effects": {
                     "success": {
-                        "reply": "There we go! No Facebook icon!",
-                        "move_to": "enable_social_bar"
+                        "reply": "That's the current clock format on this machine.",
+                        "move_to": "change_clock_format"
                     },
                     "failure": {
-                        "reply": "Hmm, I thought you'd change the value of org.gnome.shell e'enable-social-bar' to false, but it is still true. Remember to change a gsettings value you can use:\n$ gsettings set KEY VALUE"
+                        "reply": "That's not what we want. Try replacing '<our schema>' with 'org.gnome.desktop.interface' and 'clock-schema-key' with 'clock-format' in the following command `gsettings get <our-schema> <clock-schema-key>`."
```
Comment (cosimoc) [inline showmehow/lessons.json]: Should be `<clock-schema-key>`, not `'clock-schema-key'`

File: showmehow/lessons.json (hunk 3 above)
Comment (LiliC) [inline showmehow/lessons.json]: Done.

File: showmehow/lessons.json (hunk 4 above)
Comment (LiliC) [inline showmehow/lessons.json]: Done.

**Comment by LiliC** [comment]
@erikos @cosimoc addressed the comments, can you have another look. Thanks.

**Pull request merged**